src/main/java/com/couchbase/lite/javascript/JavaScriptReplicationFilterCompiler.java

```diff
@@ -0,0 +1,14 @@
+package com.couchbase.lite.javascript;
+
+import com.couchbase.lite.ReplicationFilter;
+import com.couchbase.lite.ReplicationFilterCompiler;
+
+/**
+ * Created by hideki on 10/28/15.
+ */
+public class JavaScriptReplicationFilterCompiler implements ReplicationFilterCompiler {
+    @Override
+    public ReplicationFilter compileFilterFunction(String source, String language) {
+        return new ReplicationFilterBlockRhino(source);
```

Comment:
Do we need to check what the language is and throw exception if it's not javascript?